### ISSUE TITLE
cygwin and mingw installation still buggy on windows

### ISSUE THREAD

**Issue by hohwille** [devonfw/devon-ide#209]
After fresh setup of devon-ide (beta16) I did `Open Devon Cygwin here`:
```
hohwille@laptop /cygdrive/c/projects/cool-project/workspaces
$ mvn -v
bash: mvn: command not found

hohwille@laptop /cygdrive/c/projects/cool-project/workspaces
$ devon
bash: devon: command not found
```

This is trivial to fix, but the setup however does not install `devon` on cygwin. The reason is that cygwin has its own home directory that is isolated from the regular windows home directory used by MINGW (git bash) and CMD.

In MINGW there is another strange error. I did `Git Bash here` from WindowsExplorer:
```
devon-ide has environment variables have been set for /c/projects/cool-project (workspace main)

hohwille@laptop MINGW64 /c/projects/cool-project/workspaces
$ mvn -v
bash: mvn: command not found

hohwille@laptop MINGW64 /c/projects/cool-project/workspaces
$ devon
devon-ide has environment variables have been set for /c/projects/cool-project (workspace main)

hohwille@laptop MINGW64 /c/projects/cool-project/workspaces
$ mvn -v
Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T21:00:29+02:00)
Maven home: C:\projects\cool-project\software\maven
Java version: 11.0.4, vendor: AdoptOpenJDK, runtime: C:\projects\cool-project\software\java
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```

**Comment by hohwille** [comment]
In MINGW I added some `env` statement for debugging:
```
PATH=/software/*:/c/Users/hohwille/bin:...
```
So still struggeling with `DEVON_IDE_HOME` not being set. This change was really a breaking one...

**Comment by hohwille** [comment]
Further uninstall is not consistent as it leaves some leftovers:
```
/cygdrive/c/Users/hohwille/.bashrc: line 8: devon: command not found
```

**Comment by hohwille** [comment]
Fixed with PR #210 (at least we hope all is fixed now, tests running...)
